### PR TITLE
Travis Build script

### PR DESCRIPTION
**Pull request by alancarvalho**
Issue #4.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks great

**Review by jonathonbyrdziak — APPROVED** [review]
We need to also install the checkout ext via this test sequence. but I'll get to that later.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  - sudo cat /var/log/httpd-error.log
+  - sudo cat var/log/exception.log
+  - sudo cat var/log/system.log
+  - sudo cat var/report/*
+
+after_success:
+
+ ## Tag and distribute the release
+  - sudo chmod +x $MAGE_DIR/.modman/M1-travis-deployment-files/release.sh
+  - sudo bash $MAGE_DIR/.modman/M1-travis-deployment-files/release.sh $MAGE_DIR $REPO_NAME
```
Comment (jonathonbyrdziak) [inline .travis.yml]: We need to disable the `release.sh` file for all ADDONS

**Review by jonathonbyrdziak — CHANGES_REQUESTED** [review]

**Comment by alancarvalho** [comment]
Removed.

**Review by jonathonbyrdziak — APPROVED** [review]

**Pull request merged**